packages/ui/src/primitives/tailwind/Color/index.tsx

```diff
@@ -50,7 +50,8 @@ export function ColorInput({
   textClassName,
   sketchPickerClassName
 }: ColorInputProp) {
-  const hexColor = typeof value.getHexString === 'function' ? '#' + value.getHexString() : '#000'
+  const color = new Color(value)
+  const hexColor = typeof color.getHexString === 'function' ? '#' + color.getHexString() : '#000'
```

Comment:
we can get rid of the inline conditional here, since getHexString is now guaranteed to always exist